app/templates/gulpfile.js

```diff
@@ -112,6 +122,22 @@ gulp.task('serve:dist', function () {
   });
 });
 
+gulp.task('serve:tests', function(){
+  browserSync({
+    notify: false,
+    port: 9090,
```

Comment:
In case a dev is running `gulp serve` _and_ `gulp serve:tests` simultaneously, they won't collide...